appinfo/routes.php

```diff
@@ -28,13 +34,15 @@
 		// -> columns
 		['name' => 'api1#indexTableColumns',	'url' => '/api/1/tables/{tableId}/columns', 'verb' => 'GET'],
 		['name' => 'api1#createTableColumn',	'url' => '/api/1/tables/{tableId}/columns', 'verb' => 'POST'],
+		['name' => 'api1#indexViewColumns', 'url' => '/api/1/views/{viewId}/columns', 'verb' => 'GET'],
 		['name' => 'api1#getColumn',	'url' => '/api/1/columns/{columnId}', 'verb' => 'GET'],
 		['name' => 'api1#deleteColumn',	'url' => '/api/1/columns/{columnId}', 'verb' => 'DELETE'],
 		['name' => 'api1#updateColumn',	'url' => '/api/1/columns/{columnId}', 'verb' => 'PUT'],
 		// -> rows -> table
 		['name' => 'api1#indexTableRowsSimple',	'url' => '/api/1/tables/{tableId}/rows/simple', 'verb' => 'GET'],
-		['name' => 'api1#indexTableRows',	'url' => '/api/1/tables/{tableId}/rows', 'verb' => 'GET'],
```

Comment:
Not sure if we may need to keep a compatibility endpoint (for this and other dropped ones) to just point to the base view then, as I know @stefan-niedermann is already using this API for an Android client in the works ;)

Reply:
@juliushaertl if you can give me a list of dropped endpoints I can tell you which one are problematic for the Tables Android App 🙂

Reply:
@juliushaertl / @datenangebot ping 🙂

Reply:
Given the PR has been merged, I assume that there *are* breaking changes in the API? Are the differences documented or announced somewhere?

@juliushaertl @datenangebot